src/west/app/config.py

```diff
@@ -48,6 +48,11 @@
 To set a value for <name>, type:
     west config <name> <value>
 
+To append to a value for <name>, type:
+    west config -a <name> <value>
+A value must exist in the selected configuration file in order to be able
+to append to it.
```

Comment:
Can you please add a short example? One with whitespace, quoting and `shlex.split()` to warn the user about the usual whitespace and quoting perils. Something like `west config -a build.cmake-args -- "-DCMAKE_EXPORT_COMPILE_COMMANDS=ON -DCMAKE_VERBOSE_MAKEFILE=ON"` (UNTESTED)

See https://docs.zephyrproject.org/latest/develop/west/build-flash-debug.html#permanent-cmake-arguments and 0186eade513a

Reply:
I don't mind adding the example, but:

1. there are no examples for the other actions
2. this example is particularly convoluted, perhaps not representative?

I have added two just in case in the end :)